From 36925e610901ed37fb6c52339847d283acb3bb20 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Thu, 29 Mar 2018 11:15:49 +0800
Subject: [PATCH 08/11] tweak to support external bind

Tweak the external bind to oe-core's sysroot rather than
external bind source build.

Upstream-Status: Inappropriate <oe-core specific>

Rework on 4.4.1
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 configure.ac | 34 ++++------------------------------
 1 file changed, 4 insertions(+), 30 deletions(-)

diff --git a/configure.ac b/configure.ac
index ea9f40f..2bf434d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -815,22 +815,11 @@ no)
 	if test ! -d "$use_libbind"; then
 		AC_MSG_ERROR([Cannot find bind directory at $use_libbind])
 	fi
-	if test ! -d "$use_libbind/include" -o \
-	        ! -f "$use_libbind/include/isc/buffer.h"
-	then
-		AC_MSG_ERROR([Cannot find bind includes at $use_libbind/include])
-	fi
-	if test	! -d "$use_libbind/lib" -o \
-	        \( ! -f "$use_libbind/lib/libisc.a" -a \
-		   ! -f	"$use_libbind/lib/libisc.la" \)
-	then
-		AC_MSG_ERROR([Cannot find bind libraries at $use_libbind/lib])
-	fi
 	BINDDIR="$use_libbind"
-	BINDLIBIRSDIR="$BINDDIR/lib"
-	BINDLIBDNSDIR="$BINDDIR/lib"
-	BINDLIBISCCFGDIR="$BINDDIR/lib"
-	BINDLIBISCDIR="$BINDDIR/lib"	
+	BINDLIBIRSDIR="$BINDDIR"
+	BINDLIBDNSDIR="$BINDDIR"
+	BINDLIBISCCFGDIR="$BINDDIR"
+	BINDLIBISCDIR="$BINDDIR"
 	DISTCHECK_LIBBIND_CONFIGURE_FLAG="--with-libbind=$use_libbind"
 	;;
 esac
@@ -865,21 +854,6 @@ AC_ARG_ENABLE(libtool,
 	AS_HELP_STRING([--enable-libtool],
 [use GNU libtool for dynamic shared libraries (default is no).]),
 	want_libtool="$enableval")
-
-if test "$use_libbind" != "no"; then
-	if test "$want_libtool" = "yes" -a \
-	        ! -f "$use_libbind/lib/libisc.la"
-	then
-		AC_MSG_ERROR([Cannot find dynamic libraries at $use_libbind/lib])
-	fi
-	if test "$want_libtool" = "no" -a \
-	        ! -f "$use_libbind/lib/libisc.a"
-	then
-		AC_MSG_ERROR([Cannot find static libraries at $use_libbind/lib])
-	fi
-fi
-
-
 if test "$want_libtool" = "yes"; then
 	AC_MSG_WARN([legacy configure is used but libtool is enabled. Trying to recover...])
 	# expand $ac_configure_args
-- 
1.8.3.1

